guide/17-geocoding-guides/part1_what_is_geocoding.ipynb

```diff
@@ -0,0 +1,1334 @@
+{
```

Comment:
<p>Usage of ArcGIS World Geocoding Service operations can be paid-only (e.g., when performing <code>geocode()</code>), or can be free (e.g., while using <code>suggest()</code>). Other operations can be either free or paid, depending on whether you are using them for search capabilities only, such as temporarily displaying the results, or storing the results for later use (when using <code>for_storage=True</code>). Please see the table below for more information, or check the <a href="https://developers.arcgis.com/rest/geocode/api-reference/geocoding-free-vs-paid.htm" target="_blank">REST API help doc</a>. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/830/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>